akash/paths.json

```diff
@@ -0,0 +1,204 @@
+          },
+          {
+            "path_name": "akthub",
+            "src": {
+              "chain-id": "akashnet-2",
+              "client-id": "07-tendermint-53",
+              "connection-id": "connection-29",
+              "channel-id": "channel-17",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "dst": {
+              "chain-id": "cosmoshub-4",
+              "client-id": "07-tendermint-385",
+              "connection-id": "connection-339",
+              "channel-id": "channel-184",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "tags": []
+          },
+          {
+            "path_name": "aktiov",
+            "src": {
+              "chain-id": "akashnet-2",
+              "client-id": "07-tendermint-59",
+              "connection-id": "connection-35",
+              "channel-id": "channel-23",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "dst": {
+              "chain-id": "iov-mainnet-ibc",
+              "client-id": "07-tendermint-29",
+              "connection-id": "connection-18",
+              "channel-id": "channel-6",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "tags": []
+          },
+          {
+            "path_name": "aktiris",
+            "src": {
+              "chain-id": "akashnet-2",
+              "client-id": "07-tendermint-44",
+              "connection-id": "connection-19",
+              "channel-id": "channel-11",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "dst": {
+              "chain-id": "irishub-1",
+              "client-id": "07-tendermint-23",
+              "connection-id": "connection-16",
+              "channel-id": "channel-9",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "tags": []
+          },
+          {
+            "path_name": "aktosmo",
+            "src": {
+              "chain-id": "akashnet-2",
+              "client-id": "07-tendermint-18",
+              "connection-id": "connection-11",
+              "channel-id": "channel-9",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "dst": {
+              "chain-id": "osmosis-1",
+              "client-id": "07-tendermint-0",
+              "connection-id": "connection-2",
+              "channel-id": "channel-1",
+              "port-id": "transfer",
+              "order": "unordered",
+              "version": "ics20-1"
+            },
+            "tags": []
```

Comment:
I think it would be beneficial to tag each path with an application-level meaningful tag, specifically: what is this path serving for?  I suspect in this case the path is serving a certain pool on Osmosis?

This informal is sometimes subjective or tribal knowledge, e.g., a path could the "blessed" path that Akash and Regen committed to maintain in good health and should be used when exchanging tokens between the two networks. Other times, the purpose of the path is more straightforward: it serves deposits on an DEX pool.

High-value interchain accounts will each have their own path, and should probably be maintained here in full transparency.


To make the path descriptions fully useful, I think we should clarify what is a path servicing, so that relayer operators can rely on this authoritative source of truth when they're configuring/spinning up new relayer instances.